lib/scraped/response/decorator/absolute_urls.rb

```diff
@@ -7,23 +7,31 @@ class Decorator
       class AbsoluteUrls < Decorator
         def body
           Nokogiri::HTML(super).tap do |doc|
+            @doc = doc
```

Comment:
The best way to implement this here, though, isn't entirely obvious. @chrismytton is going to ponder it…